README.md

```diff
@@ -0,0 +1,15 @@
+# Managed-notifications
+
+Notification templates for OpenShift Managed services
+
+## Usage
+
+Select the corresopnding message and send as a `POST` to 
+[api.openshift.com servicelog](https://api.openshift.com/?urls.primaryName=Service%20logs#/default/post_api_service_logs_v1_cluster_logs). 
+`CLUSTER_UUID` is a required parameter.
+
+### Examples
+
+osdctl
+
+    osdctl servicelog post -t <notificationTemplateUrl> -p CLUSTER_UUID=<UUID>
```

Comment:
this needs a cross link to where `osdctl` can be found, how to use and how to authenticate

Reply:
done